src/azure-cli/setup.py

```diff
@@ -107,7 +107,7 @@
     'azure-mgmt-netapp~=10.0.0',
     'azure-mgmt-policyinsights~=1.1.0b2',
     'azure-mgmt-privatedns~=1.0.0',
-    'azure-mgmt-rdbms~=10.2.0b7',
+    'azure-mgmt-rdbms~=10.2.0b8',
```

Comment:
It's suggested to use `==` instead of `~=` just in case there's new beta version contains breaking change

Reply:
This is OK, we will update this when the sdk releases a new version.